platform/platform-configuration/pom.xml

```diff
@@ -80,9 +82,10 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>json-smart,common-system,org.apache.felix.utils,commons-lang3</Embed-Dependency>
+                        <Embed-Dependency>
+                            json-smart,common-system,org.apache.felix.utils,commons-lang3
+                        </Embed-Dependency>
                         <Import-Package>
```

Comment:
You can remove the entire `<Import-Package>` directive now as `*` is the default.